lib/mix/tasks/stats/process.ex

```diff
@@ -9,8 +12,22 @@ defmodule Mix.Tasks.Changelog.Stats.Process do
     Mix.Task.run("app.start")
 
     case Timex.parse(List.first(args), "{YYYY}-{0M}-{D}") do
-      {:ok, time} -> Changelog.Stats.process(Timex.to_date(time))
-      {:error, _message} -> Changelog.Stats.process()
+      {:ok, time} ->
+        date = Timex.to_date(time)
+
+        Podcast.public()
+        |> Repo.all()
+        |> Enum.map(&StatsProcessor.new(%{date: date, podcast_id: &1.id}))
+        |> Oban.insert_all()
+
+      {:error, _message} ->
+        %{}
+        |> StatsProcessor.new()
+        |> Oban.insert!()
     end
+
+    results = Oban.drain_queue(queue: :scheduled, with_recursion: true, with_safety: false)
```

Comment:
The `drain_queue/1` function will execute all of the jobs in this process. The `with_recursion` option means that it will keep executing new jobs after the previous one finishes, which is required to process the sub-jobs generated by the default case. The `with_safety` flag disables "safe execution", any errors that are encountered will bubble up to the caller so you can see them from the CLI.